assets/dev/js/editor/component.js

```diff
@@ -0,0 +1,21 @@
+import Documents from './components/documents/component';
+import BrowserImport from './components/browser-import/manager';
+
+import * as commands from './commands/';
+
+export default class Component extends $e.modules.ComponentBase {
+	getNamespace() {
+		return 'editor';
```

Comment:
Do we have a `preview` component?

```suggestion
		return 'preview';
```